<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>merkezSubeEkle</class>
 <widget class="QWidget" name="merkezSubeEkle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merkez Şube Ekle</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
 	 background: #3498db;
	border-radius: 18px;
	font-family: Courier New;
	color: #ffffff;
	font-size: 15px;
	padding: 10px 20px 10px 20px;
	text-decoration: none;
    max-width:120px;
	min-width:120px;
	max-height:20px;	 
	min-height:20px;	 
}

QPushButton:hover {
	background: #3cb0fd;
	text-decoration: none;
}

QComboBox{
	border: 2.5px double #55aaff;
    border-radius: 10px;
    background: rgb(255, 255, 255);
}
QWidget{
		font-family: Courier New;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>570</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lblMerkez">
            <property name="styleSheet">
             <string notr="true">font:bold 18px </string>
            </property>
            <property name="text">
             <string>Merkez :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbMerkez">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:14px
</string>
            </property>
            <item>
             <property name="text">
              <string>Ana Merkez</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bölge Merkez 1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bölge Merkez 2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bölge Merkez 3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bölge Merkez 4</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="lblSube">
            <property name="styleSheet">
             <string notr="true">font:bold 18px </string>
            </property>
            <property name="text">
             <string>Şube :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cmbSube">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>28</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-radius:10px;
font:14px
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnEkle">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>160</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Ekle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableWidget" name="tblSubeListele">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <column>
         <property name="text">
          <string>Kod</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Durum</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Sube</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnTumu">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Tümü</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGuncelle">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Güncelle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSil">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Sil</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGeri">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>160</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Geri</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
